Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Infrastructure/Database/Migrations/20240908094351_AddSignatureText.cs

```diff
@@ -0,0 +1,41 @@
+﻿#nullable disable
+
+namespace EvolutionaryArchitecture.Fitnet.Contracts.Infrastructure.Database.Migrations;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+/// <inheritdoc />
+public partial class AddSignatureText : Migration
+{
+    /// <inheritdoc />
+    protected override void Up(MigrationBuilder migrationBuilder)
+    {
+        migrationBuilder.RenameColumn(
+            name: "SignedAt",
+            schema: "Contracts",
+            table: "Contracts",
+            newName: "Signature_Date");
+
+        migrationBuilder.AddColumn<string>(
+            name: "Signature_Text",
```

Comment:
Why with _? All the other columns follow convention SomeName

Reply:
This is generated by EF Core. When you use value object OwnOnce in configuration ef use this convention.